src/shell/js/composeui-fdc3/src/infrastructure/ComposeUITopic.ts

```diff
@@ -0,0 +1,58 @@
+/* 
+ *  Morgan Stanley makes this available to you under the Apache License,
+ *  Version 2.0 (the "License"). You may obtain a copy of the License at
+ *       http://www.apache.org/licenses/LICENSE-2.0.
+ *  See the NOTICE file distributed with this work for additional information
+ *  regarding copyright ownership. Unless required by applicable law or agreed
+ *  to in writing, software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ *  or implied. See the License for the specific language governing permissions
+ *  and limitations under the License.
+ *  
+ */
+
+import { ChannelType } from "./ChannelType";
+
+//TODO: Add more methods which returns the topics for ComposeUI's right channel
+export class ComposeUITopic {
+    private static readonly fdc3Version = "v2.0";
+    private static readonly topicRoot = `$composeui/fdc3/${this.fdc3Version}`;
+    private static readonly userChannels = '/userChannels/';
+    private static readonly appChannels = '/appChannels/';
+    private static readonly privateChannels = '/privateChannels/';
+    private static readonly broadcastSuffix = "/broadcast";
+    private static readonly getCurrentContextSuffix = "/getCurrentContext";
+    private static readonly joinUserChannelSuffix = "/joinUserChannel";
+    private static readonly getOrCreateChannelSuffix = "/getOrCreateChannel";
+
+    public static broadcast() {
+        return `${this.topicRoot}${this.broadcastSuffix}`;
```

Comment:
And with the updated strings without the separators you can construct this string by leveraging template strings like the following:
`` return `${this.topicRoot}/${this.broadcastSuffix}`; ``

And so on for the rest of the functions.

The benefit is, that you don't have to keep in mind where the separators should be for every subsrting.

So instead of doing substring1substring2, where you have to be aware of the placement of the separators, you can just simply do t as:
substring1/substring2